src/DBmysqlIterator.php

```diff
@@ -724,6 +724,8 @@ private function analyseFkey($values)
                 $fkey = $this->analyseFkey($values);
                 return $fkey . ' ' . key($condition) . ' ' . $this->analyseCrit(current($condition));
             }
+        } else if ($values instanceof QueryExpression) {
```

Comment:
I'm totally OK with taht change, but it demonstrates tests cases are missing :/

Reply:
After a second look... Well, this permits to pass quite complex raw SQL, I'm not really sure this is a good idea. 

Anyway, this is not documented (not really a problem) nor tested (it should).